Symfony/CS/FixHelper.php

```diff
@@ -0,0 +1,278 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS;
+
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\Stopwatch\Stopwatch;
+use Symfony\Component\Stopwatch\StopwatchEvent;
+
+/**
+ * Helps outputting status information when running the fix command.
+ *
+ * @author Eugene Leonovich <gen.work@gmail.com>
+ */
+class FixHelper
+{
+    /**
+     * @var OutputInterface
+     */
+    private $output;
+
+    /**
+     * @var Stopwatch
+     */
+    private $stopwatch;
+
+    /**
+     * @var bool
+     */
+    private $showProgress;
+
+    /**
+     * @var array
+     */
+    private $stats = array();
+
+    /**
+     * @param OutputInterface $output
+     * @param Stopwatch       $stopwatch
+     * @param bool|null       $showProgress
+     */
+    public function __construct(OutputInterface $output, Stopwatch $stopwatch, $showProgress = null)
+    {
+        $this->output = $output;
+        $this->stopwatch = $stopwatch;
+        $this->showProgress = (bool) $showProgress;
+    }
+
+    /**
+     * @param \Closure                 $fixFiles
+     * @param EventDispatcherInterface $eventDispatcher
+     *
+     * @return mixed
+     */
+    public function fixFiles(\Closure $fixFiles, EventDispatcherInterface $eventDispatcher)
+    {
+        $listener = array($this, 'onFileProcessed');
+
+        $eventDispatcher->addListener(FixerFileProcessedEvent::NAME, $listener);
+        $this->stopwatch->start('fixFiles');
+
+        $result = $fixFiles();
```

Comment:
Otherwise I have to write something like:

```php
$fixHelper->startFixing();
$this->fixer->fix(...);
$fixHelper->stopFixing();
```

which is not any better, or refactor `FixCommand` and break BC which I tried to avoid. But I'm open to any suggestions, if someone has a better one.